tests/blog/tests_index.py

```diff
@@ -130,7 +130,8 @@ def test_posts_timeout(self):
     #     url = "".join([self.api_url, "/media/123"])
     #
     #     responses.add(
-    #         responses.GET, url, body=requests.exceptions.Timeout(), status=504
+    #         responses.GET, url, body=requests.exceptions.Timeout(),
```

Comment:
Out of curiosity: why doesn't black catch this as well? Isn't it supposed to format stuff to fit in 80 chars?